src/net/sourceforge/kolmafia/Modifiers.java

```diff
@@ -103,8 +106,8 @@ public class Modifiers {
 
   private static final HashSet<String> numericModifiers = new HashSet<>();
 
-  public static final EnumSet<DoubleModifier> DOUBLE_MODIFIERS =
-      EnumSet.allOf(DoubleModifier.class);
+  public static final Set<DoubleModifier> DOUBLE_MODIFIERS =
+      Collections.unmodifiableSet(EnumSet.allOf(DoubleModifier.class));
```

Comment:
Not as efficient in execution or memory, but fair, given that you can't get an unmodifiable EnumSet. Yet.

Reply:
`unmodifiableSet` just creates a proxy that overrides methods that change the set to throw. In theory, I think the overhead should be minimal.